test/bigtest/interactors/BatchGroupConfigurationSettingsInteractor.js

```diff
@@ -1,7 +1,14 @@
 import {
   interactor,
+  isPresent,
 } from '@bigtest/interactor';
 
 export default @interactor class BatchGroupConfigurationSettingsInteractor {
   static defaultScope = '[data-test-batch-group-configuration-settings]';
+
+  isLoaded = isPresent('[data-test-pane-header-title]');
```

Comment:
there are might be several panes on the page with headers. I think the idea to use unique attributes